src/elements/navigation/footer.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+	import TextLine from 'elements/text/textLine.svelte';
+	import List from 'elements/list/list.svelte';
+	import ListElement from 'elements/list/listElement.svelte';
+	import SpacerOneLine from 'elements/spacer/spacerOneLine.svelte';
+	import Link from 'elements/text/link.svelte';
+	import Paragraph from 'elements/text/paragraph.svelte';
+	import LogoBig from 'elements/image/logoBig.svelte';
+
+	export let currentYear: number;
+	export let menu: [string, string][];
+
+	const startYear: number = 2024;
+
+	const numbers: number[] = Array.from(
+		{ length: currentYear + 1 - startYear },
+		(_, i) => startYear + i
+	);
+</script>
+
+<footer>
+	<div class="nav-wrapper">
+		<nav class="nav-element">
+			<TextLine classes={'white'}>Menu:</TextLine>
+			<SpacerOneLine />
+			<List>
+				{#each menu as entry}
+					<ListElement>
+						<Link classes={'standart white'} href={entry[1]}>{entry[0]}</Link>
```

Comment:
I don't know where this also has to be changed, but there's a typo.
```suggestion
						<Link classes={'standard white'} href={entry[1]}>{entry[0]}</Link>
```